src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -167,7 +167,28 @@ namespace winrt::TerminalApp::implementation
         _newTabButton.Click([weakThis{ get_weak() }](auto&&, auto&&) {
             if (auto page{ weakThis.get() })
             {
-                page->_OpenNewTab(nullptr);
+                // if alt is pressed, open a pane
+                const CoreWindow window = CoreWindow::GetForCurrentThread();
+                const auto rAltState = window.GetKeyState(VirtualKey::RightMenu);
```

Comment:
Just to make me feel better, can you set a German keyboard (or any other ones that typically have an AltGr on the right alt button) and make sure that this doesn't act super strangely? 

I don't necessarily expect it to be weird, but I do know that AltGr has caught us off-guard in the past.

Reply:
Just tried it and it splits pane as expected - nothing strange!

Reply:
Thanks for checking.